website/templates/emails/fork_failed.html.mako

```diff
@@ -0,0 +1,19 @@
+<%inherit file="notify_base.mako" />
+
+
+<%def name="content()">
+<% from website import settings %>
+<tr>
+  <td style="border-collapse: collapse;">
+
+    <h3 class="text-center" style="padding: 0;margin: 30px 0 0 0;border: none;list-style: none;font-weight: 300;text-align: center;">
+        An error has occurred, and the fork of <b>${title}</b> on The Open Science Framework was not successfully created. Please log in and try this action again. If the problem persists, please email support@osf.io.
+    </h3>
+  </td>
+</tr>
+<tr>
+  <td style="border-collapse: collapse;">
+    You can view the fork here <a href="${settings.DOMAIN + guid}">here.</a>
```

Comment:
You don't pass guid to this template so this is going to fail... which means you need tests.